### ISSUE TITLE
Methodheader for GUI communication to webserver

### ISSUE THREAD

**Issue by BenniN** [DomenicDev/pms#11]

**Comment by Colovan** [comment]
- added most method header.
- How exactly are we handling documents now? They are not defined in any class yet.
- How will we remember logged in users? should the login method return some kind of session-token that is being used for further requests?

**Comment by DomenicDev** [comment]
@Colovan Thanks for your contribution!

The option for documents to be uploaded still needs to be implemented. However, a document would just be an byte[] which will be as 'blob' stored in the database. So just define a method in the ApplicationService interface which takes the id of the doctoral student and a byte array containing the document itself. Me and @David-Geigle will take care of everything else.

Regarding the logged in users: Currently, when the client tries to log in we are parsing the credentials to the client's http context. On a successful login the client receives a session-token, the so called "JSESSIONID". This cookie is then sent on every request. 
In other words, after the (successful) login there is nothing else we need to take care of.
